docs/examples/cmark_spec_0.30.yml

```diff
@@ -5136,15 +5443,20 @@ cases:
       children:
         - type: list
           ordered: false
+          start: null
           spread: false
           children:
             - type: listItem
-              spread: true
+              spread: false
+              checked: null
```

Comment:
Should we include checked in commonmark? Probably not? 

It is part of GFM, and null is equivalent to undefined/not there:
https://github.com/syntax-tree/mdast#listitem-gfm

Reply:
this is obviously similar for `start`/`meta`/`lang`/`title`
and even if they are not included in the examples, the schema should at least accept null values, e.g. here: https://github.com/executablebooks/myst-spec/blob/a8a57d9d88c69f09c3fdc2c7ce7a1d971a223e29/schema/commonmark.schema.json#L187-L194